doc/changelog/04-tactics/13781-deprecate_micromega_options.rst

```diff
@@ -0,0 +1,3 @@
+- **Deprecated:**
+  The micromega option :flag:`Simplex` is deprecated. It is already set by default and will be removed.
```

Comment:
```suggestion
  The micromega option :flag:`Simplex`, which is currently set by default
```

`**Deprecated:**` already covers "is deprecated"
"will be removed" doesn't really add anything, that's what deprecations are about.